common/views/pages/_app.js

```diff
@@ -87,6 +107,16 @@ export default class WecoApp extends App {
       titleCase: false
     }]);
 
+    try {
+      if (document.hidden) {
+        pageVisibilityLastChanged = window.performance.now(); // in case page is opened in a new tab
+      }
+      document.addEventListener('visibilitychange', calculateHiddenTimeOnPage, false);
+      window.addEventListener('beforeunload', trackVisibleTimeOnPage);
+    } catch (error) {
+      // nada
```

Comment:
Is this for unsupported browsers?

If so, might be worth sending an event to GA, could be interesting to see how many browsers we could have tracked, but didn't.